File: Modul 4/Soal 2/PRAK402-2410817220018-ANGGRAENI-DWI-ZAHRA.c
#include <stdio.h>

int main() {
    int angka;

    printf("Masukkan Batas Maksimal: ");
    scanf("%d", &angka);

    printf(" ");
    for (int i = 1; i <= angka; i += 2) {
        printf("%d ", i);
    }
    printf("\n");

    printf(" ");
    for (int i = 2; i <= angka; i += 2) {
        printf("%d ", i);
    }
    printf("\n");

    return 0;
}

File: Modul 4/Soal 2/PRAK402-2410817220018-ANGGRAENI DWI ZAHRA.py
angka = int(input("Masukkan Batas Maksimal: "))

for i in range(1, angka+1):
    if i % 2 != 0:
        print(i, end=" ")
print()

for i in range(angka, 1, -1):
    if i % 2 == 0: 
         print(i, end=" ")